bigchaindb/core.py

```diff
@@ -293,17 +296,23 @@ def get_blocks_status_containing_tx(self, txid):
 
         # First, get information on all blocks which contain this transaction
         blocks = self.search_block_election_on_index(txid, 'transaction_id')
-
         if blocks:
             # Determine the election status of each block
-            validity = {block['id']: self.block_election_status(block) for block in blocks}
```

Comment:
Yeah, why? :smile:  

> As with list comprehensions, an explicit for loop can always be
    used (and in fact was the only way to do it in earlier versions of
    Python).  But as with list comprehensions, dict comprehensions can
    provide a more syntactically succinct idiom that the traditional
    for loop. -- [PEP 274](https://www.python.org/dev/peps/pep-0274/)